docs/UserGuide.md

```diff
@@ -150,27 +187,33 @@ Unmarks a specified event in Paidlancers.
 **Format**: `unmark INDEX`
 
 - Unmarks the event at the specified `INDEX`.
+    - The `INDEX` refers to the index number in the displayed events list.
+    - The `INDEX` must be a positive integer 1, 2, 3, …
 
 **Example**:
-- `unmark 2` unmarks the `2nd event`.
+- `unmark 2`
+  - This command will unmark the 2nd event in the event list.
 
 ### Create new Event: `newevent` <a id = "create-new-event"></a>
 
 Creates a new event
 
-**Format**: `newevent n/NAME p/rate a/ADDRESS ds/START_TIME de/END_TIME [t/TAG]…`
+**Format**: `newevent n/NAME r/rate a/ADDRESS ds/START_TIME de/END_TIME [t/TAG]…`
 
 - Both `START_TIME` and `END_TIME` must have the format `dd-MM-yyyy HH:mm`.
 
 **Example**:
-- `newevent n/DJ at wedding p/100 a/311, Clementi Ave 2, #02-25 ds/11-03-2023 11:00 de/11-03-2023 17:00 t/friends t/dj`
+- `newevent n/DJ at wedding r/100 a/311, Clementi Ave 2, #02-25 ds/11-03-2023 11:00 de/11-03-2023 17:00 t/friends t/dj`
+  - This command will create a new event named `DJ at wedding` with rate `100` at address `311, Clementi Ave 2, #02-25` from `11-03-2023 11:00` to `11-03-2023 17:00` with tags `friends` and `dj`.
 
 ### List all Events: `listevent` <a id = "list-all-events"></a>
```

Comment:
Perhaps `listevent` could be changed to `list`?

Reply:
Done!